zeebe/logstreams/src/test/java/io/camunda/zeebe/logstreams/impl/log/SequencerTest.java

```diff
@@ -30,7 +31,7 @@ final class SequencerTest {
   @Test
   void notifiesConsumerOnWrite() {
     // given
-    final var sequencer = new Sequencer(0, 16, new SequencerMetrics(1));
+    final var sequencer = new Sequencer(0, 16, new SequencerMetrics(new SimpleMeterRegistry()));
```

Comment:
:thought_balloon:  I usually create a registry in the tests class with `@AutoClose` or `AutoCloseResource`. Probably an overkill, but using these patterns even in test code is helpful for consistency and to learn/remember about scopes and resources

Reply:
You're right, I was going a bit fast :see_no_evil: